### DISCUSSION TITLE
Relocating Organizations Between Sentry Instances

### DISCUSSION THREAD

**Discussion by chadwhitacre** [getsentry/sentry#61450] [Launch Feedback]
As [announced during Launch Week](https://blog.sentry.io/sentrys-eu-data-region-now-in-early-access/), we are building better tools to move your existing organization data from one Sentry instance to another, following closely behind the [rollout of our new EU region](https://github.com/getsentry/sentry/discussions/49564). This discussion thread is to track the rollout of this “org relocation” feature.

To start off, we are looking for our first few customers to test this new functionality with us. The requirement is that you:

1. have an up-to-date version of self-hosted (>= 2023.11.0), with
2. test data in a non-production organization that can safely go into our multi-region test environment (we don't want your customer PII in our test region).

We will work with you closely to go through the process and address any issues that come up. If you are interested, please drop a comment here or [email me](mailto:chadwhitacre@sentry.io). We will likely also reach out to folks who are on [the waitlist](https://sentry.io/trust/privacy/#data-residency-form) so if that applies to you, feel free to let me know. Thanks!

**Comment by johannesnagl** [comment]
@chadwhitacre happy to work with you on that topic. I've been in talks with @gauthamcs for a while already - so he has all of my contact data!

**Reply by chadwhitacre** [reply]
Thank you @johannesnagl! I will follow up with @gauthamcs internally and go from there. 👍

**Comment by chadwhitacre** [comment]
[We're live](https://docs.sentry.io/product/accounts/migration/)! 🙌
